resources/views/main_screen.blade.php

```diff
@@ -0,0 +1,29 @@
+@extends('layout')
+
+@section('head')
+    <link rel="stylesheet" href="{{asset('stylesheets/configStyle.css')}}">
+    <link rel="stylesheet" href="{{asset('stylesheets/mainScreenStyle.css')}}">
```

Comment:
Als deze standaard styling ook op elke pagina wordt gebruikt, dan zou ik die ook in de `layout` plaatsen.

Reply:
Zelfde verhaal als eerder. Die file is puur bedoeld voor de twee schermen welke ik heb gemaakt. Als iemand anders hem ook wil gebruiken voor een nieuw scherm kan hij die in de layout gooien wat mij betreft